java/client/src/test/java/glide/api/models/commands/options/BitFieldOptionsTests.java

```diff
@@ -0,0 +1,42 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands.options;
+
+import static glide.api.models.commands.bitmap.BitFieldOptions.createBitFieldArgs;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+import glide.api.models.commands.bitmap.BitFieldOptions.BitFieldGet;
+import glide.api.models.commands.bitmap.BitFieldOptions.BitFieldIncrby;
+import glide.api.models.commands.bitmap.BitFieldOptions.BitFieldOverflow;
+import glide.api.models.commands.bitmap.BitFieldOptions.BitFieldSet;
+import glide.api.models.commands.bitmap.BitFieldOptions.BitFieldSubCommands;
+import glide.api.models.commands.bitmap.BitFieldOptions.Offset;
+import glide.api.models.commands.bitmap.BitFieldOptions.OffsetMultiplier;
+import glide.api.models.commands.bitmap.BitFieldOptions.SignedEncoding;
+import glide.api.models.commands.bitmap.BitFieldOptions.UnsignedEncoding;
+import org.junit.jupiter.api.Test;
+
+public class BitFieldOptionsTests {
```

Comment:
is it necessary to create a whole new file for one test?

Reply:
Wasn't sure where it fit because it was inconsistent with the tests in `RedisClientTest`.